### ISSUE TITLE
Missing alt text on Design Tokens page

### ISSUE THREAD

**Issue by adorayi** [cds-snc/design-gc-conception#680]
Missing alt text
On the Design Tokens page, all images are marked as decorative even though they are informative and communicate information. Empty alt text should not be used on informative visuals. 

E.g., the anatomy of tokens is communicated through images. These images do not have text alternatives. 
<img width="554" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMXdYQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4bbdede85e3053fa1baeb25a75a5762cdb672307/image.png" alt="image.png" />

**Comment by SmartMouthWords** [comment]
Reviewed alt text updates from @ethanWallace 
- Added suggested changes in new column to the [sheet Ethan created](https://docs.google.com/spreadsheets/d/1FHBEau_Bep6C_FU2Duf58MbDUlY6v_5Bjm9kEGSplaA/edit#gid=0)
- Brought these to content critique 20-Feb-2024 and integrated feedback.
